exp-player/addon/components/exp-overview/component.js

```diff
@@ -114,7 +114,12 @@ export default ExpFrameBaseComponent.extend(Validations, {
         var questions = this.get('questions');
         var responses = {};
         for (var i=0; i < questions.length; i++) {
-            responses[i] = questions[i].value;
+            if (i === 0) {
+                // Convert value to int bc select-input returns a string (e.g. "16" --> 16)
+                responses[i] = parseInt(questions[i].value);
```

Comment:
That's a bit gnarly, but we can improve it in the future.

As I recall, this field has a default value (null): https://github.com/CenterForOpenScience/exp-addons/blob/a3d18733534d6a35db449f276ec3294948037e37/exp-player/addon/components/select-input/template.hbs#L2-L2

Are we comfortable with the default serialization (`NaN`) that would result? It doesn't throw an error, but there is a slight assumption that this is always an integer.